src/ResourceManagement/TrafficManager/TrafficManagerManagement/Properties/AssemblyInfo.cs

```diff
@@ -20,8 +20,8 @@
 [assembly: AssemblyTitle("Microsoft Azure Traffic Manager Management Library")]
 [assembly: AssemblyDescription("Provides Microsoft Azure Traffic Manager management functions for managing the Microsoft Azure Traffic Manager service.")]
 
-[assembly: AssemblyVersion("1.0.0.0")]
-[assembly: AssemblyFileVersion("1.0.0.0")]
+[assembly: AssemblyVersion("1.1.0.0")]
```

Comment:
AssemblyVersion shopuld remain at 1.0.0.0 to maintain backward compatibility, AssemblyFileVersion should match the package version (as it does).